File: January/22-01-23/OopEvidence/src/q1_details/Student.java

package q1_details;

public class Student {
    int id;
    String email;
    int round;
    String address;

    public String showAddress(){
        return "My address is : " + address;
    }
    
    public Student(){
        
    }
    public Student(int id, String email, int round, String address){
        this.id = id;
        this.email = email;
        this.round = round;
        this.address = address;
    }
}

File: july/8-07-24 - search by id/flutter/lib/student.dart
class Student {
  int? id;
  String? name;
  String? email;
  String? round;

  Student(
      {required this.id,
        required this.name,
        required this.email,
        required this.round});

  factory Student.fromJson(Map<String, dynamic> json) => Student(
    id: json["id"],
    name: json["name"],
    email: json["email"],
    round: json["round"],
  );

  Map<String, dynamic> toJson() {
    return {
      "id": id,
      "name": name,
      "email": email,
      "round": round,
    };
  }
}
